charts/finrem-ns/values.preview.template.yaml

```diff
@@ -2,7 +2,3 @@ java:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
-  environment:
-    AUTH_PROVIDER_SERVICE_CLIENT_BASEURL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-    REFORM_ENVIRONMENT: aat
-  aadIdentityName:
```

Comment:
this is still needed, its unsetting it as we don't use it on preview yet

Reply:
restored